v2/src/modules/address/ModuleAddressRecentNfts.vue

```diff
@@ -67,10 +73,14 @@ const props = defineProps({
  * NFT Data
  -------------------------*/
 
-const { result: resultBalance, loading: loadingBalance } = useGetOwnersNftTokensQuery(
+const {
+    result: resultBalance,
+    loading: loadingBalance,
+    fetchMore
+} = useGetOwnersNftTokensQuery(
     () => ({
         address: props.addressHash,
-        limit: props.isOverview ? 9 : 50
+        limit: props.isOverview ? 9 : ITEMS_PER_PAGE
```

Comment:
i think this is the only case when we should not use ITEMS_PER_PAGE. We should use multiple of 12 ( so that there is no empty space on different layouts. I think we should use max of 48 item.